### PR TITLE
Javascript Capstone Project

### PR DESCRIPTION
**Pull request by EmekaEmmanuel**
## Javascript Capstone Project
- Fetch data from TVMAZE and Involvement API.
- Display all movie cards.
- Display all likes, comment and reservation attached to a movie.
-  Get the total length of movies, likes, comments and reservation 
- Webpack, ES6 modules and Jset test.

### CODE REVIEW

File: src/api/getAllShows.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
```
Comment (Ghiftee) [inline src/api/getAllShows.js]: - [x] Let's try to not disable linters as it helps us write cleaner code.

File: src/config.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */
```
Comment (Ghiftee) [inline src/config.js]: - [x] Let's try to not disable linters as it helps us write cleaner code.

File: src/modules/comment/InvolvementAPI.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+const InvolvementAPI = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/';
+const InvolvementID = 'eOrcCP3sdE79tBRIZ6tU';
```
Comment (Ghiftee) [inline src/modules/comment/InvolvementAPI.js]: - [x] I noticed you defined this involvement API in several places, I suggest that you define it once like you did in ```config.js``` and import it wherever you need it like you did in ```getAllLikes.js```. This would help your code fulfill the DRY principle.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @EmekaEmmanuel, @zewdu444, @jugosack 👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ Tests pass
✔️ Gitflow is used
✔️ No linter error

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

- [x] Good job implementing the likes and reservations for each movie, however, when I add a comment to the first movie on the screen, it gets recorded in the second movie. Kindly fix this.
- [x] Kindly ensure your counter functions are covered with unit tests using Jest. Make sure that you have tests for all edge cases, for instance, a scenario in which there are 0 items on the page (counter function should return 0)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by OyePriscilla — APPROVED** [review]
Hi @EmekaEmmanuel, @zewdu444 and @jugosack 😄, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✔️Git flow observed 👍
✔️App worked properly 👍
✔️Made required changes 👍

## Optional suggestions

- [optional] Let's take sometime to work more on your README file to make it looks professional. Kindly add **one or more future feature(s)**, and kindly format almost all the sections (**getting started, show your support, acknowledgement, built with ...**) according to the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md)  

- [optional] The UI/UX design of your app is not responsive, It would be great if you can work on the mobile version (most especially the pop-up modals for comment and reservation) as well for a great UI/UX. 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
